## Metagenomics-SummerProject ##

# Title: Metagenomics through time, in a crop wild relative of tomato
Abstract
## Introduction: 
The phyllosphere microbiome can plays a central role in plant health, yet little is known about it in Solanum chilense, a drought- and pathogen-tolerant wild tomato relative. Herbarium specimens offer an opportunity to study historical microbiomes but may reflect storage-related changes. This study compared the phyllosphere microbiomes of S. chilense from historical herbarium specimens and contemporary field-collected samples.
## Methods: 
Eighteen dried leaf samples were analyzed (9 herbarium specimens (1938–2005) and 9 contemporary samples (2022)). Raw Illumina reads underwent adapter trimming, host read removal, and deduplication before taxonomic classification with Kraken2. Diversity analyses in QIIME2 included Shannon entropy for alpha diversity and Bray–Curtis dissimilarities for beta diversity, with statistical testing via Kruskal–Wallis and PERMANOVA.
## Results: 
While the dominant phylum in all samples included Pseudomonata, Ascomycota, and Bacillota, herbarium samples demonstrated a clear increase of Ascomycota in them compared to the field-collected samples. Alpha diversity did not differ significantly between groups, but beta diversity showed clear compositional separation (PERMANOVA p < 0.05). Field samples had more even, balanced communities, while herbarium specimens were often dominated by a few taxa.
## Conclusion: 
Although microbial richness remained stable, herbarium samples showed considerable shifts in taxonomic composition, probably due to postmortem colonization and storage effects. These findings highlight both the potential and limitations of using preserved plant material for historical microbiome reconstruction.

## HYPOTHESIS
Microbial communities in historical herbarium specimens will differ from those in modern samples due to postmortem colonization and long-term storage conditions. Applying a contaminant-filtering pipeline will reduce technical noise and reveal biologically relevant taxa that reflect genuine temporal shifts in plant-associated microbiota.

## References
1. Bieker VC, Sánchez Barreiro F, Rasmussen JA, Brunier M, Wales N, Martin MD. Metagenomic analysis of historical herbarium specimens reveals a postmortem microbial community. Mol Ecol Resour. 2020;20(5):1206–19.
2. Waters C, Hurt C, Krosnick S. Looking to the past to inform the future: What eDNA from herbarium specimens can tell us about plant–animal interactions. Appl Plant Sci. 2025;13(2):e11633.
3. Bakker FT, Hemerik L. Herbarium DNA degradation – Falling to pieces non-randomly. BAUHINIA – Z Basl Bot Ges. 2023 Dec 31;29:63–74.
4. Runge P, Ventura F, Kemen E, Stam R. Distinct Phyllosphere Microbiome of Wild Tomato Species in Central Peru upon Dysbiosis. Microb Ecol. 2023 Jan 1;85(1):168–83.
